app/scripts/controllers/authentication/authentication-controller.test.ts

```diff
@@ -254,6 +254,7 @@ function createAuthenticationMessenger() {
   return messenger.getRestricted({
     name: 'AuthenticationController',
     allowedActions: [`SnapController:handleRequest`],
+    allowedEvents: [],
```

Comment:
Prior to `@metamask/base-controller` 5.0.0, `allowedEvents` was optional if no events were allowlisted; now this argument is required. See changelog: https://github.com/MetaMask/core/blob/main/packages/base-controller/CHANGELOG.md#changed-2